src/app/online-tracker/online-tracker.component.html

```diff
@@ -0,0 +1,6 @@
+<p *ngIf="this.onlineTrackerService.isOnline" class="text-bold"><i class="fa fa-circle text-success"></i>
+  <span></span>
+</p>
+<p *ngIf="!this.onlineTrackerService.isOnline" class="text-bold"><i class="fa fa-circle text-danger"></i>
```

Comment:
You should move this to a helper method in the component which should then call the service